packages/sonarwhal/src/lib/utils/npm.ts

```diff
@@ -12,21 +12,43 @@ import * as logger from './logging';
 const debug: debug.IDebugger = d(__filename);
 const npmLoadAsync = promisify(npm.load);
 
-
-const packageExists = () => {
+const getPackageContent = (): string => {
     const packagePath: string = path.join(process.cwd(), 'package.json');
+    let packageContent: string;
+
+    try {
+        packageContent = fs.readFileSync(packagePath, 'utf8'); // eslint-disable-line no-sync
+    } catch (err) {
+        // package.json` doesn't exist.
+        return null;
+    }
 
-    return fs.existsSync(packagePath); // eslint-disable-line no-sync
+    return packageContent;
```

Comment:
Maybe you can do a parse here already. Maybe use `misc.loadJSONFile`, or directly `request('package.json')`